bentoml/server/bento_api_server.py

```diff
@@ -141,12 +142,17 @@ def feedback_view_func(bento_service):
         User send feedback along with the request Id. It will be stored and
         ready for further process.
         """
-        if request.content_type != "application/json":
+        if request.method == 'GET':
+            params = parse.parse_qs(parse.urlparse(request.url).query)
+            # This is a bit hacky as parse_qs returs a list for every query
+            data = {i: j[-1] for (i, j) in params.items()}
+        elif request.content_type != "application/json":
```

Comment:
If this is something that is not desirable, we can just remove GET support for `/feedback`

Reply:
Hi @Korusuke, it looks like "GET /feedback" does not really work before this PR and I think it is ok to just remove it. "POST" makes more sense here

Reply:
Okey, cools!